### PR TITLE
Proxy pulp/deb to pulpcore support

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
I think i did this right?

File: manifests/init.pp [hunk 1]
```diff
@@ -40,6 +40,8 @@
 #
 # $pulp_ca_cert::                       Absolute path to PEM encoded CA certificate file, used by Pulp to validate the identity of the broker using SSL.
 #
+# $proxy_pulp_deb_to_pulpcore::        Proxy /pulp/deb to Pulpcore at /pulp/content
```
Comment (ekohl) [inline manifests/init.pp:43]: Alignment looks off. Not sure if it's Github rendering it off
```suggestion
# $proxy_pulp_deb_to_pulpcore::         Proxy /pulp/deb to Pulpcore at /pulp/content
```

File: manifests/init.pp [hunk 2]
```diff
@@ -416,6 +420,9 @@
     class { 'pulpcore::plugin::rpm':
       use_pulp2_content_route => $proxy_pulp_yum_to_pulpcore,
     }
+    class { 'pulpcore::plugin::deb':
+      use_pulp2_content_route => $proxy_pulp_deb_to_pulpcore,
```
Comment (ekohl) [inline manifests/init.pp:424]: This requires a corresponding PR to https://github.com/theforeman/puppet-pulpcore/blob/master/manifests/plugin/deb.pp to allow this.

Comment (jlsherrill) [reply manifests/init.pp:424]: ahh thanks!

**Comment by ekohl** [comment]
This depends on https://github.com/theforeman/puppet-pulpcore/pull/145. We should also take note to increase the minimum version to one that includes the change once merged.

**Review by wbclark — COMMENTED** [review]
@jlsherrill could you update metadata.json here to require puppet-pulpcore version 2.1 where support for the pulp2 content route is available in the deb plugin ?

**Review by ekohl — APPROVED** [review]

**Comment by jlsherrill** [comment]
@wbclark does this look okay now?

**Review by wbclark — APPROVED** [review]
Yes, it looks good to me. Thanks @jlsherrill !

**Pull request merged**